package.json

```diff
@@ -196,7 +196,7 @@
     "@newrelic/aws-sdk": "^7.3.0",
     "@newrelic/koa": "^9.1.0",
     "@newrelic/ritm": "^7.2.0",
-    "@newrelic/security-agent": "^1.1.1",
+    "@newrelic/security-agent": "^1.2.0",
```

Comment:
```suggestion
    "@newrelic/security-agent": "1.2.0",
```

We'd like to pin this dependency going forward. Please remove the `^` qualifier.

Reply:
this is ok but @sumitsuthar we don't need PRs when you release unless it's a semver major.  The `^1.x.x` will include all releases in 1.x

Reply:
Got it.